itunes-connect.csv

```diff
@@ -0,0 +1,27 @@
+Name,Aware
+Privacy Policy URL,
+Bundle ID,com.github.josh.Aware
+SKU,Aware1
+Primary Language,English
+Primary Category,Productivity
+Secondary Category,
+Price,USD 0.99 (Tier 1)
+Availability,Available in all territories
+Volume Purchase Program,Available with a volume discount for educational institutions
+Bitcode Auto-Recompilation,Use bitcode auto-recompilation
+Screenshot 1,
+Screenshot 2,
+Screenshot 3,
+Screenshot 4,
+Screenshot 5,
+Description,
```

Comment:
Should include our brief description line. But it seems like you get up to 4000 characters here. So we could include another paragraph if that seems helpful.